metadata_layer.py

```diff
@@ -220,23 +231,27 @@ def apply(self, destination, replace=None):
             if not k in destination.extra:
                 destination.extra[k] = v
 
-        if self.lc != destination.lc:
```

Comment:
This and the changes below are actually ~kinda a big deal~. Instead of replacing information wholesale, we only replace contributor information that exists and leave the rest. I'd like to wait and discuss this with @daryanypl, who probably has a better sense of where contributors are being overwritten in the system.